modules/azure/monitoring_action_group/main.tf

```diff
@@ -0,0 +1,41 @@
+terraform {
+  required_version = "~> 1.3"
+
+  required_providers {
+    azurerm = {
+      source  = "hashicorp/azurerm"
+      version = "~> 3.40"
+    }
+  }
+
+  backend "azurerm" {}
+}
+
+provider "azurerm" {
+  features {}
+}
+
+resource "azurerm_monitor_action_group" "action_group" {
+  name                = var.name
+  resource_group_name = var.resource_group_name
+  short_name          = var.short_name
+  enabled             = var.enabled
+
+  dynamic "email_receiver" {
+    for_each = var.email_receiver == null ? [] : [1]
```

Comment:
Is it an idea to support multiple e-mail adresses and make it a list of strings?

Reply:
The same is true for the webhook_uri this can also be a list of strings

Reply:
Well, that's a good question, I was moving this logic from another file. But since the variable for email is not an array, I assumed this was just a fancy way to create 0 or 1.
I can update it to this if we think it is a bit clearer
**count = var.email_receiver == null ? 0 : 1**
Should be equivalent, right?

Reply:
```suggestion
    for_each = var.email_receivers
```